### PR TITLE
update for biocore

### LINKED ISSUE #31: Validate links from tutorials
There are a number of static links scattered in #30 , after the PR is merged into master we should go through and double check that all of them work.

### PR DESCRIPTION
**Pull request by cameronmartino**
Fix #31 and edit for biocore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ There are several ways to run CTF through gemelli. Here we provide a tutorial of
 
 ### Tutorials with QIIME2
 
-* [QIIME2 Command Line Interface](https://github.com/cameronmartino/gemelli/blob/master/ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md)
-* [QIIME2 API](https://nbviewer.jupyter.org/github.com/cameronmartino/gemelli/blob/master/ipynb/tutorials/IBD-Tutorial-QIIME2-API.ipynb)
+* [QIIME2 Command Line Interface](https://github.com/biocore/gemelli/blob/master/ipynb/tutorials/IBD-Tutorial-QIIME2-CLI.md)
```
Comment (gwarmstrong) [inline README.md:23]: I think there are a number of other links like this in the jupyter notebooks that should be changed as well, e.g.,

https://github.com/cameronmartino/gemelli/blob/825beea36eeda4e7e9b0bd988c1b21c5d26f6e2a/ipynb/tutorials/IBD-Tutorial-QIIME2-API.ipynb#L18-L22

Comment (cameronmartino) [reply README.md:23]: Good catch - thanks!

**Comment by gwarmstrong** [comment]
Okay that **looks** good. First issue in biocore might be to go through and check all the links, once things are moved over.

**Comment by cameronmartino** [comment]
@gwarmstrong Definitely. Thaks!

**Pull request merged**